reactor-core/src/main/java/reactor/core/publisher/Flux.java

```diff
@@ -7799,6 +7799,13 @@ public int getPrefetch() {
 		return onAssembly(new FluxScanSeed<>(this, initial, accumulator));
 	}
 
+	//TODO javadoc
+	public final <R> Flux<R> scope(ScopeOptions options, Function<Flux<T>, Flux<R>> transform) {
+		return Flux.deferWithContext(original -> transform
```

Comment:
nit: can we please use non-short (`original -> {}`) lambda here, for readability? :)